bundles/org.openhab.binding.haywardomnilogic/README.md

```diff
@@ -65,14 +65,55 @@ Hayward OmniLogic Connection Parameters:
-| chlorError            | Number               | Chlorinator error                                        |      R     |
-| chlorAlert            | String               | Chlorinator alert                                        |      R     |
+| chlorError            | String               | Chlorinator error bit array                              |      R     |
+| chlorAlert            | String               | Chlorinator alert bit array                              |      R     |
 | chlorAvgSaltLevel     | Number:Dimensionless | Chlorinator average salt level in Part per Million (ppm) |      R     |
 | chlorInstantSaltLevel | Number:Dimensionless | Chlorinator instant salt level in Part per Million (ppm) |      R     |
-| chlorStatus           | Number               | Chlorinator K1/K2 relay status                           |      R     |
+| chlorStatus           | String               | Chlorinator status bit array                             |      R     |
+
+**Item Types Number:Dimensionless should have the units (i.e. %, ppm) defined in the Unit metadata 
+
+### Chlorinator Error Bit Array
+
+|Bits  |Value        				 		    |Description                     
+|------|---------------------------------------|--------------------------------|
+|1:0   |00 = OK<br>  01 = Short<br> 10 = Open  								|Current Sensor			  		| 
+|3:2   |00 = OK<br>  01 = Short<br> 10 = Open  								|Voltage Sensor             	| 
+|5:4   |00 = OK<br>  01 = Short<br> 10 = Open  								|Cell Temp Sensor               | 
+|7:6   |00 = OK<br>  01 = Short<br> 10 = Open  								|Board Temp Sensor              | 
+|9:8   |00 = OK<br>  01 = Short<br> 10 = Open  								|K1 Relay                 		| 
+|11:10 |00 = OK<br>  01 = Short<br> 10 = Open  								|K2 Relay                 		| 
+|13:12 |00 = OK<br>  01 = Type<br> 10 = Authentication <br> 11 = Comm Loss 	|Cell Errors                	| 
+|14    |0									   								|Aquarite PCB Error             | 
+
+### Chlorinator Alert Bit Array
+
+|Bits  |Value        				 		    |Description                     
+|------|--------------------------------------------------------------------|-------------------------------|
+|1:0   |00 = OK<br>  01 = Salt Low<br> 10 = Salt too Low					|Low salt				  		| 
+|2     |0 = OK<br>  1 = High  												|High Current            		| 
+|3     |0 = OK<br>  1 = Low					 								|Low Voltage              		| 
+|5:4   |00 = OK<br>  01 = Low<br> 10 = Scaleback<br> 11 = High				|Cell Water Temp          		| 
+|7:6   |00 = OK<br>  01 = High<br> 10 = Clearing							|Board Temp                		| 
+|8     |0																	|Not Used	                	| 
+|10:9  |0									   								|Not Used				        | 
+|12:11 |00 = OK<br>  01 = Clean				   								|Cell Cleaning/Runtime          | 
+
+### Chlorinator Status Bit Array
+
+|Bits  |Value        				 		    |Description                     
```

Comment:
Looks like a "|" is missing at the end of this line.
A proper alignment of all lines would also be appreciated.